cider/tests/functionality/CiderNonEqualLoopJoinTest.cpp

```diff
@@ -22,164 +22,215 @@
 #include <gtest/gtest.h>
 #include "tests/utils/CiderTestBase.h"
 
-class CiderNonEqualJoinTest : public CiderJoinTestBase {
+// TODO: (spevenhe) For not null query, if set NOT NULL on column ddl, the final project
+// will go to cider_agg_id_proj_xx() instead of cider_agg_id_proj_xx_nullable(), So the
+// returned null values are incorrect.
```

Comment:
why results are incorrect?

Reply:
Even if we create NOT NULL column, in left join cases, there should be null results when left table failed to find a match in right table. But current columnvar codegen will not go to null handle part if we set column for NOT NULL.